### DISCUSSION TITLE
Supplying input parameter OpenCV.Net.ColorConversion

### DISCUSSION THREAD

**Discussion by edmundchong** [bonsai-rx/bonsai#1610] [Q&A] [answered]
Hi all, 

I'm trying to use the SLEAP node PredictSinglePose, and one of the parameters that it can take is `ColorConversion (System.Nullable<OpenCV.Net.ColorConversion>)`. I'm trying to set this parameter at runtime, by reading from a text file.

How do I convert from a string read from a text file (e.g ."RGB2Gray"), to the correct format for this?
Additionally, if in some cases, if no color conversion is desired, what should the supplied string input be? 

Thanks!

**Comment by bruno-f-cruz** [comment, marked as answer]
Without any custom code, i think the easiest would be to use a `Condition` + `PropertySource`:

![image](https://github.com/bonsai-rx/bonsai/assets/7049351/4d6e7189-df06-4a8f-8cb9-5495a352e94d)

**Comment by edmundchong** [comment]
Works well, thanks!!
